README.md

```diff
@@ -41,14 +40,14 @@ A small sample is available as a playground.
 
 An example of instantiation as follow with the fallback to the pin code with some text descriptions to display for the user.
 
-```
+``` cs
```

Comment:
does that work?
It think it's usually ```csharp

Reply:
It works, I used the preview feature on VSCode.